diff --git a/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/Makefile b/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/Makefile
index 3893a87..027ed13 100644
--- a/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/Makefile
+++ b/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/Makefile
@@ -1,5 +1,6 @@
 EXTRA_CFLAGS += $(USER_EXTRA_CFLAGS)
 EXTRA_CFLAGS += -Wno-implicit-fallthrough
+EXTRA_CFLAGS += -Wno-int-conversion
 #EXTRA_CFLAGS += -Wno-unused-variable
 
 RWNX_VERS_NUM := 6.4.3.0
